### ISSUE TITLE
High sample count crashes IBLBaker

### ISSUE THREAD

**Issue by remcohuijser** [derkreature/IBLBaker#12]
When you choose a high number of samples (anything higher than 512) IBLBaker stops. In debug there is no exception given.

**Comment by remcohuijser** [comment]
I tried to disconnect the sample count from the samples per frame so that the samples are spread out over a number of frames. This seems to work as IBLBaker takes a couple of frames to calculate the end result and you can actually see the intermediate result being updated.

I did notice something strange though: it appears that the result from the previous pass are not correctly combined with the next pass. As a result result becomes 'smeared' along the edges (see image below):
![image](https://user-images.githubusercontent.com/12158789/48200491-9cfc9f80-e35f-11e8-8504-1f97b43984af.png)

Because of this, the resulting map is not usable :(

**Comment by derkreature** [comment]
Yeah, I imagine there is a bug in offsetting the sample Id in regards to hammersley point generation. I’ll take a look.

**Comment by derkreature** [comment]
Are you just hitting TDR timeouts?

**Comment by remcohuijser** [comment]
I am not sure to be honest. Any way I can check?

**Comment by derkreature** [comment]
I’ll take a look at this over the holiday break.

**Comment by remcohuijser** [comment]
Much appreciated!

**Comment by derkreature** [comment]
I am getting to this.
